pkg/ioutils/sort.go

```diff
@@ -14,4 +14,26 @@
  * limitations under the License.
  */
 
-package main
+package ioutils
+
+import "sort"
+
+// SortUnique sorts data which results in distinct elements in data in sorted order.
+func SortUnique(data sort.Interface) {
```

Comment:
SortUnique is supposed to remove duplicate entries. This function is swapping already sorted entries. What is the outcome?